crates/daemon/src/fapolicyd.rs

```diff
@@ -32,7 +35,7 @@ pub enum Version {
 }
 
 /// send signal to fapolicyd FIFO pipe to reload the trust database
-pub fn reload_databases() -> Result<(), Error> {
+pub fn signal_trust_reload() -> Result<(), Error> {
```

Comment:
There is no value in keeping this.  We cannot assume to write anything to disk without first taking the daemon down. If the daemon comes down there is no need for signal.